.github/workflows/erlang.yml

```diff
@@ -9,22 +9,24 @@ on:
       - master
 jobs:
   ci:
-    name: Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
-    runs-on: ${{matrix.os}}
+    name: OTP ${{matrix.otp}}
+    runs-on: ubuntu-latest
+    # enable this to test all OTP versions at once regardless the errors
+    # continue-on-error: true
     container:
-      image: erlang:${{matrix.otp_vsn}}
+      image: erlang:${{matrix.otp}}
     strategy:
       matrix:
-        otp_vsn: ["19.0", "19.3",
```

Comment:
OTP 19 not in the official supported Docker image tags. 
See https://hub.docker.com/_/erlang/